### PR TITLE
Refresh token periodically through credential fetcher

### PR DESCRIPTION
**Pull request by liminw**
Refresh token periodically through credential fetcher.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[x] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

File: security/pkg/nodeagent/cache/secretcache.go [hunk 1]
```diff
@@ -578,27 +578,27 @@ func (sc *SecretCache) rotate(updateRootFlag bool) {
 			return true
 		}
 
+		if sc.configOptions.CredFetcher != nil {
+			// Refresh token through credential fetcher.
+			cacheLog.Infof("%s getting a new token through credential fetcher", logPrefix)
+			t, err := sc.configOptions.CredFetcher.GetPlatformCredential()
+			if err != nil {
+				cacheLog.Errorf("failed to get credential token: %v", err)
```
Comment (JimmyCYJ) [inline security/pkg/nodeagent/cache/secretcache.go:586]: cacheLog.Errorf("credential fetcher failed to get credential token: %v", err)

Comment (liminw) [reply security/pkg/nodeagent/cache/secretcache.go]: Done.

**Pull request merged**